server/src/test/java/org/opentosca/toscana/plugins/kubernetes/docker/image/DockerImageBuilderTest.java

```diff
@@ -0,0 +1,57 @@
+package org.opentosca.toscana.plugins.kubernetes.docker.image;
+
+import java.io.File;
+
+import org.opentosca.toscana.plugins.kubernetes.docker.BaseDockerfileTest;
+
+import com.spotify.docker.client.DefaultDockerClient;
+import com.spotify.docker.client.DockerClient;
+import org.junit.Before;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assume.assumeTrue;
+
+public class DockerImageBuilderTest extends BaseDockerfileTest {
```

Comment:
Tests in this class take over 1 min on my machine. This is too long (at least for basic unit tests).
Extends from BaseJUnitTest, but is integration test. I'm aware that we currently don't have a proper (fast/slow flags is not sufficient imho, its a hack) mechanism for integration tests. We can't merge this PR until we have a solution for this, though.

Reply:
I have a idea to fix this (at least for now)